src/super_gradients/training/datasets/pose_estimation_datasets/abstract_pose_estimation_dataset.py

```diff
@@ -50,6 +51,18 @@ def __init__(
             load_sample_fn=self.load_random_sample,
         )
         self.num_joints = num_joints
+
+        # Explicitly convert edge_links, keypoint_colors and edge_colors to lists of tuples
+        # This is necessary to ensure ListConfig objects do not leak to these properties
+        # and from there - to checkpoint's state_dict.
+        # Otherwise, through ListConfig instances a whole configuration file will leak to state_dict
+        # and torch.load will attempt to unpickle lot of unnecessary classes.
+        edge_links = [(int(from_idx), int(to_idx)) for from_idx, to_idx in edge_links]
```

Comment:
`OmegaConf.to_container()` cannot do this convertion? 
If not, it could be good to have a function that converts any DictConfig/ListConfig to native python objects, like here where you want `List[Tuple[int, int]]`.

Reply:
Actually this snippet solves two issues at the same time:
1) Converts to primitive types 
2) Ensures that edge_links passed in the correct format (E.g array as N,2 shape or something that can be represented as an array of pairs)
So I think here is justified.
Unfortunately OmegaConf.to_container cannot take list or dict as input (It crashes when fed a primitive type)